org.vaccine.app.json

```diff
@@ -26,17 +30,18 @@
       "sources": [
         {
           "type": "git",
-          "url": "git://git.gnome.org/libgee"
+          "url": "https://gitlab.gnome.org/giuliobenetti/libgee.git"
         }
       ]
     },
     {
       "name": "gtksourceview",
+      "buildsystem": "meson",
       "sources": [
         {
           "type": "git",
-          "url": "git://git.gnome.org/gtksourceview",
-          "branch": "gnome-3-24"
+          "url": "https://gitlab.gnome.org/taozuhong/gtksourceview.git",
```

Comment:
Shouldn't this be `https://gitlab.gnome.org/GNOME/gtksourceview.git`?

Reply:
Yep, fixed